apps/examples/elf/wifiapp/wifiapp.c

```diff
@@ -68,6 +68,7 @@ int main(int argc, char **argv)
 		case 'x':
 			printf("Test will be finished.\n");
 			is_testing = false;
+			break;
```

Comment:
Please revert this line. This was merged with commit https://github.com/Samsung/TizenRT/commit/c1fcc954f0627b5165282c8fc96a312ca856c7a2

Reply:
modify